### PR TITLE
Adding large GPMSA verification example

### PR DESCRIPTION
**Pull request by dmcdougall**
Replaces #607.

### CODE REVIEW

File: examples/gp/linear_verif/gpmsa_linear_verif_pdf.C [hunk 1]
```diff
@@ -0,0 +1,614 @@
...
+  }
+
+  // Experiment observation error (also needs to be scaled by
+  // simulation variance)
+  for (unsigned int i = 0; i < numExperiments; i++)
+    (*experimentMat)(i, i) = std::pow(0.05, 2.0);
+
+  // Read in data and store the standard deviation of the simulation
+  // data (ignored for now).
+  readData("sim_scalar.dat", "y_exp_scalar.txt", simulationScenarios, paramVecs,
```
Comment (roystgnr) [inline examples/gp/linear_verif/gpmsa_linear_verif_pdf.C]: Are these files included in the PR anywhere?

Comment (roystgnr) [reply examples/gp/linear_verif/gpmsa_linear_verif_pdf.C]: Wait, never mind, those are the files that were merged in #580.

**Comment by dmcdougall** [comment]
@roystgnr Ok I think I'm done here too.  Let me know if you have any comments.

**Comment by dmcdougall** [comment]
@briadam This is the pull request where I destroyed all your hard work.  I'll resurrect it.

**Comment by briadam** [comment]
Hardly destroyed... just morphed it into a necessary prerequisite test. ;-)  

Honestly, you'll probably do better to just write an inverse problem departing from scalar_pdf_large without looking at what I did as you'll likely then immediately see the issues I ran into with the initial positions and proposal covariance.  I can see having to scale the proposal covariance as we discussed today, but it's possible at this point (given all the work you've done) that if you take the initial point at the mean and proposal covariance to be the prior variance, you may fare better than I did, and it'll just work.

**Comment by dmcdougall** [comment]
Well, it'll be a pull request in a couple of minutes and then you can tell me what to do with it :)

**Pull request merged**